gallery/gallery/lib/studies/fortnightly/shared.dart

```diff
@@ -0,0 +1,532 @@
+              ),
+            ),
+          ],
+        )
+      ],
+    );
+  }
+}
+
+List<Widget> buildStockItems(BuildContext context) {
+  Widget articleDivider = Container(
+    margin: EdgeInsets.symmetric(vertical: 16),
+    color: Colors.black.withOpacity(0.07),
+    height: 1,
+  );
+
+  return <Widget>[
+    SizedBox(
+      width: double.infinity,
+      child: Image.asset(
+        'assets/fortnightly/fortnightly_chart.png',
+        fit: BoxFit.contain,
+        excludeFromSemantics: true,
+      ),
+    ),
+    articleDivider,
+    StockItem(
+      ticker: 'DIJA',
+      price: '7,031.21',
+      percent: -0.48,
+    ),
+    articleDivider,
+    StockItem(
+      ticker: 'SP',
+      price: '1,967.84',
+      percent: 0.00,
+    ),
+    articleDivider,
+    StockItem(
+      ticker: 'Nasdaq',
+      price: '6,211.46',
+      percent: 0.52,
+    ),
+    articleDivider,
+    StockItem(
+      ticker: 'Nikkei',
+      price: '5,891',
+      percent: 1.16,
+    ),
+    articleDivider,
+    StockItem(
+      ticker: 'DJ Total',
+      price: '89.02',
+      percent: 0.80,
+    ),
+    articleDivider,
+  ];
+}
+
+class VideoPreview extends StatelessWidget {
+  VideoPreview({this.data, this.time});
+
+  final ArticleData data;
+  final String time;
+
+  @override
+  Widget build(BuildContext context) {
+    TextTheme textTheme = Theme.of(context).textTheme;
+
+    return Column(
+      crossAxisAlignment: CrossAxisAlignment.start,
+      children: [
+        SizedBox(
+          width: double.infinity,
+          child: Image.asset(
+            data.imageUrl,
+            fit: BoxFit.contain,
+            excludeFromSemantics: true,
+          ),
+        ),
+        SizedBox(height: 4),
+        Row(
+          children: [
+            Text(data.category, style: textTheme.subhead),
+            Spacer(),
+            Text(time, style: textTheme.body2)
+          ],
+        ),
+        SizedBox(height: 4),
+        Text(data.title, style: textTheme.headline.copyWith(fontSize: 16)),
+      ],
+    );
+  }
+}
+
+List<Widget> buildVideoPreviewItems(BuildContext context) {
+  return <Widget>[
+    VideoPreview(
+      data: ArticleData(
+        imageUrl: 'assets/fortnightly/fortnightly_feminists.jpg',
+        category: GalleryLocalizations.of(context)
+            .fortnightlyMenuPolitics
+            .toUpperCase(),
+        title: GalleryLocalizations.of(context).fortnightlyHeadlineFeminists,
+      ),
+      time: '2:31',
+    ),
+    SizedBox(height: 32),
+    VideoPreview(
+      data: ArticleData(
+        imageUrl: 'assets/fortnightly/fortnightly_bees.jpg',
+        category:
+            GalleryLocalizations.of(context).fortnightlyMenuUS.toUpperCase(),
+        title: GalleryLocalizations.of(context).fortnightlyHeadlineBees,
+      ),
+      time: '1:37',
+    ),
+  ];
+}
+
+ThemeData buildTheme(BuildContext context) {
+  TextTheme textTheme = Theme.of(context).textTheme;
+  return ThemeData(
```

Comment:
TODO: Use google_fonts now that #299 is merged

Reply:
Done